Eastern-Research-Group/mywaterway
# Feature/636 dependency updates

## Related Issues:
* [HMW-636](https://jira.epa.gov/browse/HMW-636)
* [HMW-637](https://jira.epa.gov/browse/HMW-637)

## Main Changes:
* Updated changelog.
* Updated app level dependencies.
* Updated client level dependencies.
  * Skipping React 18 again, due to lack of time.
  * Skipping styled components, due to lack of time.
  * Skipping react-sticky-box as there is a bug in the `2.0.3` and `2.0.4` versions. [More info](https://github.com/codecks-io/react-sticky-box/issues/87).
* Skipped server level dependencies as there weren't any major updates.

## Steps To Test:
1. Verify changelog updates look good. Note: Prettier did remove some spaces which is th reason for changes to older text.
2. Test features related to the package updates.
